orttraining/orttraining/core/optimizer/qdq_fusion.cc

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "orttraining/core/optimizer/qdq_fusion.h"
+
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+
+namespace onnxruntime {
+
+namespace {
+
+NodeArg& ReplaceZeroPointInitializer(Graph& graph, const std::string& zero_point_initializer_name) {
+  // Replace the original zero point initializer with the float zero point initializer
+
+  auto zero_point_tensor_int = graph.GetInitializer(zero_point_initializer_name, true);
+  ONNX_NAMESPACE::TensorProto zero_point_tensor_float;
+  zero_point_tensor_float.set_name(graph.GenerateNodeArgName(zero_point_tensor_int->name()));
+  zero_point_tensor_float.set_data_type(ONNX_NAMESPACE::TensorProto_DataType_FLOAT);
+  for (const auto val : zero_point_tensor_int->int32_data()) {
```

Comment:
why is this int32? Don't you need to check whether d_type is int8 or uint8 and fetch data accordingly?

Reply:
OK, since int32_data implementation is in ONNX, I feel we'd better to use this with care in case they changed some implementation details later.